nav2_waypoint_follower/include/nav2_waypoint_follower/waypoint_follower.hpp

```diff
@@ -43,6 +43,18 @@ enum class ActionStatus
   SUCCEEDED = 3
 };
 
+struct MissedWaypoint
```

Comment:
We had a discussion about adding a new nav2_msgs/MissedWaypoint message for this so that it can be directly correlated more easily in the message. Was that dropped for a technical reason?